api/pkg/db/migration/202010061403_rename_name_to_agent_name_in_user.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"github.com/jinzhu/gorm"
+	"gopkg.in/gormigrate.v1"
+
+	"github.com/tektoncd/hub/api/gen/log"
+)
+
+func renameNameColumnToAgentNameInUserTable(log *log.Logger) *gormigrate.Migration {
```

Comment:
@PuneetPunamiya is there a way we can test the migration? Have you thought of a way we can ensure that the migration will run fine against currently deployed version?